logic/transaction.js

```diff
@@ -850,13 +802,14 @@ Transaction.prototype.dbSave = function (transaction) {
 		throw 'Unknown transaction type ' + transaction.type;
 	}
 
-	var senderPublicKey, signature, signSignature, requesterPublicKey;
+	var senderPublicKey, signature, signSignature, requesterPublicKey, recipientPublicKey;
 
 	try {
 		senderPublicKey = Buffer.from(transaction.senderPublicKey, 'hex');
 		signature = Buffer.from(transaction.signature, 'hex');
 		signSignature = transaction.signSignature ? Buffer.from(transaction.signSignature, 'hex') : null;
 		requesterPublicKey = transaction.requesterPublicKey ? Buffer.from(transaction.requesterPublicKey, 'hex') : null;
+		recipientPublicKey = transaction.recipientPublicKey ? Buffer.from(transaction.recipientPublicKey, 'hex') : null;
```

Comment:
Is that should be part of this PR? Do we verify somewhere if recipientId match recipientPublicKey?

Reply:
recipient_public_key is a new column on transactions table to enable sending by publickey at a later date. some tasks todo around getbytes to enable that functionality.